charts/caraml/values.yaml

```diff
@@ -1,3 +1,49 @@
 # Default values for caraml.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+
+mlp:
+  # -- To enable/disable MLP chart installation.
+  enabled: true
+  postgresql:
+    # -- To enable/disable MLP specific postgres
+    enabled: false
```

Comment:
how do we get mlp connected to postgresql?

Reply:
By default, In this chart, CaraML creates a Postgres `caraml-postgresql`( which internally creates it's secret `caraml-postgresql-secret`, MLP has [helper functions](https://github.com/caraml-dev/helm-charts/pull/7/files#diff-b602ae30aece9d733f9ec49bed1e03b2bf40b78dbedc47d1763501304e0a2886R63) to choose the right database. They first check if mlp specific database is enabled, if so they use the names corresponding to that database. else if external db is enabled connect to it, else fallback to the release level database. 

These helper functions help choose the right db, username, and password secret.